src/Akeneo/Platform/Job/back/Application/FindJobTypes/FindJobTypesHandler.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Platform\Job\Application\FindJobTypes;
```

Comment:
SearchJobExecution is singular we should be consistent
```suggestion
namespace Akeneo\Platform\Job\Application\FindJobType;
```